src/types/marketplace.ts

```diff
@@ -0,0 +1 @@
+export type Marketplace = 'gcm' | 'aws' | 'azure'
```

Comment:
Above, we're looking for `account.marketplace !== 'Zuora'`, but here `Zuora` isn't an option.  And in the spec we looked at for `/me`, we said that `marketplace` can be `null`.

Let's figure out which direction we want to go and be consistent with it. Otherwise, we're gonna get a `null` marketplace and try to compare that with `Zuora` and bad things will happen.